app/models/chargeback_container_image.rb

```diff
@@ -22,6 +22,10 @@ class ChargebackContainerImage < Chargeback
     :total_cost                 => :float,
   )
 
+  def self.refresh_dynamic_metric_columns
```

Comment:
This method should be put in the base model and removed from here and the container project model.